.github/workflows/community-pr.yml

```diff
@@ -28,16 +28,17 @@ jobs:
                 authorAssociation
               }
             }
-          }' -f org=$ORGANIZATION -f repo=$REPOSITORY -f pull="$PULL_REQUEST_NUMBER" -f label="$LABEL_NAME" > data.json
+          }' -f org=$ORGANIZATION -f repo=$REPOSITORY -F pull=$PULL_REQUEST_NUMBER -f label="$LABEL_NAME" > data.json
```

Comment:
This fixes the following error:

> Variable $pull of type Int! was provided invalid value

The GitHub CLI uses `-f` for "raw fields" which are always strings, and `-F` for typed fields.